appcues/src/main/java/com/appcues/statemachine/Transitions.kt

```diff
@@ -71,7 +71,7 @@ internal interface Transitions {
                     // in different groups we want to wait for StartStep action from AppcuesViewModel
                     val response = CompletableDeferred<ResultOf<State, Error>>()
                     Transition(
-                        state = EndingStep(experience, flatStepIndex) {
+                        state = EndingStep(experience, flatStepIndex, true) {
```

Comment:
the case here and on line 84 - we're ending a step because we are getting a StartStep action for a new step - in those cases, we always consider the step we're ending to be completed, as we're moving elsewhere.